src/components/features/about-section/components/discover-button.tsx

```
@@ -0,0 +1,11 @@
+import { Button } from "@components/ui/button";
+import { ArrowRight } from "lucide-react";
```

ESLint

[error] 'ArrowRight' is defined but never used.

(@typescript-eslint/no-unused-vars)